ramls/budgets.raml

```diff
@@ -67,4 +67,32 @@ resourceTypes:
         collection-get:
           exampleCollection: !include acq-models/mod-finance/examples/budget_expense_class_totals_collection.sample
           schemaCollection: budgetExpenseClassTotalsCollection
+    /recalculate:
+      displayName: Recalculate Budget
+      description: Recalculate the budget based on transactions.
+      post:
+        is: [ validate ]
+        description: Recalculate the budget
+        responses:
+          200:
```

Comment:
for post request, 201 is more accure response code
![image](https://github.com/folio-org/mod-finance/assets/113523904/d5dcc764-9455-47c8-aa4b-ebd73ce74920)

Reply:
I think it should be 204(no content) response. because:

- 204 No Content is suitable when the server successfully processes the request, but there's no need to return any data in the response body.
- 201 Created is more appropriate when the request results in the creation of a new resource, which may not align with triggering a budget recalculation.

Reply:
Sounds good to me 👍

Reply:
One more reqest is that it shouldn't led to confusion with DELETE request. No content is usually more appropriate in delete resources (i have seen many places of folio).